docs/detections/rules-coverage.asciidoc

```diff
@@ -1,9 +1,51 @@
 [[rules-coverage]]
 == MITRE ATT&CK® coverage
 
-:frontmatter-description: 
+:frontmatter-description: Review your current coverage of MITRE ATT&CK® tactics and techniques, based on installed rules.
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
-:frontmatter-tags-user-goals: [manage]
+:frontmatter-tags-user-goals: [manage, analyze, visualize]
 
-This is a placeholder for future documentation.
+The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.
```

Comment:
```suggestion
The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your active detection rules. This includes both Elastic prebuilt rules and custom rules.
```

Reply:
I think we need to be extra precise here, to explain exactly which rules are included on this page, and what actions (install and enable) the user needs to do to include rules.